.bazelci/update_workspace_to_deps_heads.sh

```diff
@@ -13,12 +13,6 @@ set -euo pipefail
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")\
 \
 git_repository(\
-\    name = "bazel_skylib",\
```

Comment:
since tulsi doesn't actually reference skylib I think it's fair to not mess with this here, this caused issue because rules_apple patches this repo, we could apply the same patch here but that just feels leaky